### DISCUSSION TITLE
terminate Octernship

### DISCUSSION THREAD

**Discussion by Kishan1835** [community/community#50634] [GitHub Education] [answered]
### Terminate

**Question**

### Body

How to terminate an Octernship or add a new one

**Comment by queenofcorgis** [comment, marked as answer]
Welcome to the GitHub Community, @Kishan1835, we're happy you're here! You are more likely to get a useful response if you are posting your questions in the applicable category and are explicit about what your question entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
